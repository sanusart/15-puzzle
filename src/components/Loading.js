import React from 'react';

const Loading = () => <div>l<span className="spin" style={{ fontSize: '15px' }}>&empty;</span>ading</div>;

export default Loading;
