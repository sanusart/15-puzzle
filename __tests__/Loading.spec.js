import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../src/components/Loading';

it('Snapshot for <Loading /> should match', () => {
  const loading = renderer.create(<Loading />);
  const json = loading.toJSON();
  expect(json).toMatchSnapshot();
});
