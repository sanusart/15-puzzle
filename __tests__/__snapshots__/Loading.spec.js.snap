exports[`test Snapshot for <Loading /> should match 1`] = `
<div>
  l
  <span
    className="spin"
    style={
      Object {
        "fontSize": "15px",
      }
    }>
    âˆ…
  </span>
  ading
</div>
`;
